Userland/Applications/SpaceAnalyzer/main.cpp

```diff
@@ -416,7 +379,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
                 continue;
             }
 
-            const SpaceAnalyzer::TreeMapNode* node = treemapwidget.path_node(k);
+            const TreeNode* node = treemapwidget.path_node(k);
```

Comment:
Nit-pick: Can you move the `const` to the right side while you're editing this line?